<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Loging</class>
 <widget class="QWidget" name="Loging">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Авторизация в системе документо оборота</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius:30px;
background-color: rgb(52, 167, 255);</string>
  </property>
  <widget class="QLineEdit" name="login">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic [UKWN]</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(62, 167, 255);
border-style: solid;
border-width: 2px;
border-color: blue;
border-radius:16px;
padding:16px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic [UKWN]</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(62, 167, 255);
border-style: solid;
border-width: 2px;
border-color: blue;
border-radius:16px;
padding:16px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(92, 167, 255);
border-style: solid;
border-width: 1px;
border-color: blue;
border-radius:20px;
padding:16px;</string>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;URW Gothic [UKWN]&quot;;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Система документооборота для&lt;/p&gt;&lt;p&gt;ГБПОУ «Грозненский государственный колледж информационных технологий»&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="show_password">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:16px;
background-color: none;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="audentifaceit_errorLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>321</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(52, 167, 255);</string>
   </property>
   <property name="text">
    <string>Введенные данные некорректны</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
